rca/project_styleguide/templates/patterns/atoms/modal-launcher/modal-launcher.html

```diff
@@ -5,7 +5,7 @@
     <a href="{{ item.link }}" class="modal-launcher">
 {% endif %}
     <div class="modal-launcher__container">
-        <div class="modal-launcher__label">
+        <div class="modal-launcher__label" id="{{ modal_aria_label }}">
```

Comment:
should this be an `aria-label` rather than an `id`?

Reply:
it should be `id`, just a confusing variable name